datarepo-clienttests/src/main/java/runner/config/SpecificationInterface.java

```diff
@@ -2,12 +2,19 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.concurrent.TimeUnit;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public interface SpecificationInterface {
   Logger logger = LoggerFactory.getLogger(SpecificationInterface.class);
 
+  int totalNumberToRun = 1;
+  int numberToRunInParallel = 1;
+  long expectedTimeForEach = 300;
+  TimeUnit expectedTimeForEachUnitObj = TimeUnit.SECONDS;
+  String description = "";
```

Comment:
I disagree with moving these parameters into the interface. This interface is used by other specification classes also, including test user and service account, where these parameters don't make sense.